src/Models/WishlistItem.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Rapidez\MultipleWishlist\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class WishlistItem extends Model
+{
+    protected $table = 'wishlist_item';
```

Comment:
This can be removed right?

Reply:
You'd think so, and that's what I thought as well. However, as it turns out Laravel will only look for the plural table `wishlist_items` rather than `wishlist_item`.